### DISCUSSION TITLE
Add a private option to /openurl (or a new command)

### DISCUSSION THREAD

**Discussion by Jaffythethird** [Chatterino/chatterino2#5147] [Ideas] [answered]
Currently, /openurl works great for things like translating on the spot. However, there are some options I wish it had, like being able to open in an incognito browser and/or being able to define properties about the window it pops up (like size, location, current window, new window, etc). The latter seems a bit ambitious, but I think it would be nice to be able to use my translate command without it flooding my browsing/search history.

I imagine that one issue would be that different browsers refer to private windows as different things (Chrome is Incognito Mode, Firefox is Private Window, etc.)

**Comment by pajlada** [comment, marked as answer]
EDIT: This is now available in the latest [nightly build](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) - thanks for the feature request!

I made https://github.com/Chatterino/chatterino2/pull/5149 which should solve your issue
You can now add `--private` or `--incognito` to force private/incognito mode
Or, if you have your setting set to use incognito links by default, you can use `--no-private` or `--no-incognito` to open the link in normal mode.
`--private` and `--incognito` are aliases, and use our underlying code to figure out private mode browsing
